spec/features/idv/in_person_spec.rb

```diff
@@ -449,16 +462,43 @@
         ),
       )
 
+      expect(page).to have_content(
+        I18n.t(
+          'in_person_proofing.form.state_id.errors.unsupported_chars',
+          char_list: '$',
+        ),
+      )
+
+      expect(page).to have_content(
+        I18n.t(
+          'in_person_proofing.form.state_id.errors.unsupported_chars',
+          char_list: '@, ^',
+        ),
+      )
+
+      expect(page).to have_content(
+        I18n.t(
+          'in_person_proofing.form.state_id.errors.unsupported_chars',
+          char_list: '!, 3',
+        ),
+      )
```

Comment:
These messages will differ.

```suggestion
      expect(page).to have_content(
        I18n.t(
          'in_person_proofing.form.address.errors.unsupported_chars',
          char_list: '$',
        ),
      )

      expect(page).to have_content(
        I18n.t(
          'in_person_proofing.form.address.errors.unsupported_chars',
          char_list: '@, ^',
        ),
      )

      expect(page).to have_content(
        I18n.t(
          'in_person_proofing.form.address.errors.unsupported_chars',
          char_list: '!, 3',
        ),
      )
```